commands/FaunaCommands.js

```diff
@@ -0,0 +1,129 @@
+const FQLXCommands = require("./FQLXCommands");
+const FQL4DeployCommand = require("./FQL4DeployCommand");
+const FQL4RemoveCommand = require("./FQL4RemoveCommand");
+
+class FaunaCommands {
+  command = {
```

Comment:
why are these (`command` and `hooks`) assigning into global scope?

@macmv do you know? these are using hte same pattern the original serverless did - wondering if you ran into this over teh summer and had a take here since its night where Lucas is.

Reply:
So I don't know if there was history here, but moving them into the constructor works great, and most of the serverless plugin dev documentation also assigns them in the constructor.

PR has been updated